config/management/src/lib.rs

```diff
@@ -295,47 +295,114 @@ pub mod tests {
     #[test]
     #[ignore]
     fn test_end_to_end() {
+        // Each identity works in their own namespace
+        // Alice, Bob, and Carol are an operators, implicitly mapped 1:1 with Owner.
+        // Dave is the association.
+        // Each user will upload their contents to *_ns + "shared"
+        // Common is used by the technical staff for coordination.
```

Comment:
I see where you're going with this test, but some refactoring and splitting it up into smaller chunks/digestible functions would really make it much easier to reason about